### PR TITLE
[Doc] Update BE_configuration.md

### PR DESCRIPTION
**Pull request by nshangyiming**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/zh/administration/BE_configuration.md [hunk 1]
```diff
@@ -409,13 +409,13 @@ curl -XPOST http://be_host:http_port/api/update_config?configuration_item=value
 
 #### storage_flood_stage_usage_percent
 
-- 含义：如果空间使用率超过该值且剩余空间小于 `storage_flood_stage_left_capacity_bytes`，会拒绝 Load 和 Restore 作业。
+- 含义：如果空间使用率超过该值且剩余空间小于 `storage_flood_stage_left_capacity_bytes`，会拒绝 Load 和 Restore 作业。需要同步修改 FE 配置 `storage_usage_hard_limit_percent`。
```
Comment (evelynzhaojie) [inline docs/zh/administration/BE_configuration.md:412]: 建议：先解释一下这个参数的含义，再写”如果 xxx“

Comment (evelynzhaojie) [reply docs/zh/administration/BE_configuration.md:412]: 比如：BE 磁盘空间使用率。如果 xxx

File: docs/zh/administration/BE_configuration.md [hunk 2]
```diff
@@ -409,13 +409,13 @@ curl -XPOST http://be_host:http_port/api/update_config?configuration_item=value
...
 - 单位：%
 - 默认值：95
 
 #### storage_flood_stage_left_capacity_bytes
 
-- 含义：如果剩余空间小于该值且空间使用率超过 `storage_flood_stage_usage_percent`，会拒绝 Load 和 Restore 作业，默认 100GB。
+- 含义：如果剩余空间小于该值且空间使用率超过 `storage_flood_stage_usage_percent`，会拒绝 Load 和 Restore 作业，默认 100GB。需要同步修改 FE 配置 `storage_usage_hard_limit_reserve_bytes`。
```
Comment (evelynzhaojie) [inline docs/zh/administration/BE_configuration.md:418]: 含义：BE 磁盘剩余空间。如果 xxx

File: docs/zh/administration/BE_configuration.md (hunk 1 above)
Comment (nshangyiming) [reply docs/zh/administration/BE_configuration.md:412]: 含义：BE 存储目录整体磁盘空间使用率的硬限制阈值。

File: docs/zh/administration/BE_configuration.md (hunk 2 above)
Comment (nshangyiming) [reply docs/zh/administration/BE_configuration.md:418]: 含义：BE 存储目录整体磁盘剩余空间的硬限制阈值。

**Review by EsoragotoSpirit — DISMISSED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**